ReactAndroid/src/main/java/com/facebook/react/modules/fresco/FrescoModule.java

```diff
@@ -154,7 +154,7 @@ private static ImagePipelineConfig getDefaultConfig(ReactContext context) {
     HashSet<RequestListener> requestListeners = new HashSet<>();
     requestListeners.add(new SystraceRequestListener());
 
-    OkHttpClient client = OkHttpClientProvider.createClient();
+    OkHttpClient client = new DefaultOkHttpProvider().get();
```

Comment:
I haven't looked in to the details of how Fresco is using this client. But I guess, Fresco might need its own customised client configuration. 

For now, I will add a comment in `MainPackageConfig`. Will take a look at this later.  Would be great if someone with Fresco context can share some light.